### DISCUSSION TITLE
Security-check

### DISCUSSION THREAD

**Discussion by PlatinDig** [dani-garcia/vaultwarden#2680] [General]
Hi, I scanned the docker image using trivy. 
Here are the results. 
Most of the problems are related to the MariaDB version.

vaultwarden/server (debian 11.4)

Total: 370 (UNKNOWN: 1, LOW: 133, MEDIUM: 43, HIGH: 178, CRITICAL: 15)

**Comment by BlackDex** [comment]
Yes, what do you suggest? It's bad practice to use external repo's within container images, so that is kinda out of the question for me. It also makes building the image more difficult.

If you want less trivy issues, i suggest to use the Alpine images.
